MoreLinq.Test/AssertException.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace MoreLinq.Test
+{
+    public class AssertThrows : Assert
+    {
+        public static void ArgumentNull(string argumentName, TestDelegate code)
+        {
+            ThrowsArgumentException<ArgumentNullException>(argumentName, code);
+        }
+
+        public static void Argument(string argumentName, TestDelegate code)
+        {
+            ThrowsArgumentException<ArgumentException>(argumentName, code);
+        }
+
+        public static void OutOfRange(string argumentName, TestDelegate code)
+        {
+            ThrowsArgumentException<ArgumentOutOfRangeException>(argumentName, code);
+        }
+
+        public static void InvalidOperation(TestDelegate code)
```

Comment:
I don't find it's worth wrapping every exception like `InvalidOperationException`, `SequenceException` and `Exception`. These should be reverted back to simply using `Assert.Throws`.